x/autopilot/keeper/liquidstake.go

```diff
@@ -72,12 +72,47 @@ func (k Keeper) RunLiquidStake(ctx sdk.Context, addr sdk.AccAddress, token sdk.C
 	}
 
 	msgServer := stakeibckeeper.NewMsgServerImpl(k.stakeibcKeeper)
-	_, err := msgServer.LiquidStake(
+	result, err := msgServer.LiquidStake(
 		sdk.WrapSDKContext(ctx),
 		msg,
 	)
 	if err != nil {
-		return errorsmod.Wrapf(sdkerrors.ErrInsufficientFunds, err.Error())
+		return errorsmod.Wrapf(err, err.Error())
 	}
-	return nil
+
+	if packetMetadata.IbcReceiver == "" {
+		return nil
+	}
+
+	hostZone, err := k.stakeibcKeeper.GetHostZoneFromHostDenom(ctx, token.Denom)
+	if err != nil {
+		return err
+	}
+
+	return k.IBCTransferStAsset(ctx, result.StToken, addr.String(), hostZone, packetMetadata)
+}
+
+func (k Keeper) IBCTransferStAsset(ctx sdk.Context, stAsset sdk.Coin, sender string, hostZone *stakeibctypes.HostZone, packetMetadata types.StakeibcPacketMetadata) error {
+	ibcTransferTimeoutNanos := k.stakeibcKeeper.GetParam(ctx, stakeibctypes.KeyIBCTransferTimeoutNanos)
+	timeoutTimestamp := uint64(ctx.BlockTime().UnixNano()) + ibcTransferTimeoutNanos
+	channelId := packetMetadata.TransferChannel
```

Comment:
Would it be safer to not take the channel as input from the user and restrict forwarding to chains Stride supports as host zones? In most cases, passing it in would be fine because it'd be handled on the frontend. But I could imagine integrations who want to use autopilot getting it wrong and consequently losing tokens on wrong channels / chains.

The downside of restricting to only host zones is that we can't LS + forward to chains like Neutron or other defi hubs we don't support for liquid staking yet. I lean towards keeping it as is, but curious to hear your thoughts on this.

Reply:
+1 to the keeping current setup! 

It's definitely more error prone, but feels like a must have to be able to send to non-host zone chains.

Unfortunately, I do think this reinforces the fact that we should might need a more robust fallback strategy in the case of an ack failure though

Reply:
we discussed this on a call and decided on keeping the current setup! marking resolved